sdk/python/foundation-models/system/finetune/image-classification/multiclass-classification/hftransformers-fridgeobjects-multiclass-classification.ipynb

```diff
@@ -220,7 +216,9 @@
     "aml_registry_model_name = \"microsoft-beit-base-patch16-224-pt22k-ft22k\"\n",
     "model_version = \"1\"\n",
```

Comment:
@gauravrajguru  - I mean here. We are specifying the version manually, right? I meant, let's bring this in sync with how we have it in other notebook, extract the latest version and use that.

Reply:
Updated it.